src/workflows/airqo_etl_utils/meta_data_utils.py

```
@@ -62,6 +68,90 @@
+        metadata_method = {
+            MetaDataType.DEVICES: MetaDataUtils.extract_devices,
+            MetaDataType.SITES: MetaDataUtils.extract_sites,
+        }
+        unique_id = "device_id" if metadata_type == MetaDataType.DEVICES else "site_id"
+
+        # Retrieve metadata table and columns
+        metadata_table, cols = DataUtils._get_metadata_table(
+            MetaDataType.DATAQUALITYCHECKS, metadata_type
+        )
+        entities = metadata_method.get(metadata_type)()
+        entities = entities[(entities.network == "airqo") & (entities.deployed == True)]
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `entities.deployed:` for truth checks

Replace with `entities.deployed`

(E712)